charts/virtual-kubelet/templates/secrets.yaml

```diff
@@ -1,12 +1,22 @@
+{{- $ca := genCA "virtual-kubelet-ca" 3650 }}
+{{- $cn := printf "%s-virtual-kubelet-apiserver" .Release.Name }}
+{{- $altName1 := printf "%s-virtual-kubelet-apiserver.%s" .Release.Name .Release.Namespace }}
+{{- $altName2 := printf "%s-cvirtual-kubelet-apiserver.%s.svc" .Release.Name .Release.Namespace }}
```

Comment:
How com these are all generated here instead of only when they are needed below?

Reply:
updated :)